xapian-core/weight/tfidfweight.cc

```diff
@@ -63,8 +63,8 @@ TfIdfWeight::TfIdfWeight(const std::string &normals, double slope, double delta)
     : normalizations(normals), param_slope(slope), param_delta(delta)
 {
     if (normalizations.length() != 3 ||
-	!strchr("nbslPL", normalizations[0]) ||
-	!strchr("ntpfsP", normalizations[1]) ||
+	!strchr("nbslPLaS", normalizations[0]) ||
+	!strchr("ntpfsPEGl", normalizations[1]) ||
```

Comment:
You've not documented any of your new normalisations in the API docs (see `include/xapian/weight.h` for where all the currently supported ones are defined).